### PR TITLE
fix(TBD-11039): Dataproc Spark job fails with java.lang.NoSuchMethodError

### PR DESCRIPTION
**Pull request by firefryy**
…starting

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml [hunk 1]
```diff
@@ -63,21 +63,21 @@
         <!-- 1.62.0 -->
         <libraryNeeded
 		    context="plugin:org.talend.hadoop.distribution.dataproc14"
-		    id="google-cloud-storage-1.73.0"
-		    name="google-cloud-storage-1.73.0.jar"
-		    mvn_uri="mvn:com.google.cloud/google-cloud-storage/1.73.0">
+		    id="google-cloud-storage-1.2.1"
```
Comment (Redwene) [inline main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:66]: why did you downgrade the version ?

Comment (firefryy) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:66]: hive-exec jar (which we can not change) use a lower guava version of those jars so we have a nosuchmethod issue, so i had to downgrade the version of those jars

**Review by Redwene — APPROVED** [review]
lgtm

File: main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml (hunk 1 above)
Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:66]: Have you checked that guava versions used by hive-exec and google-cloud-storage-1.2.1 are the same ?

Comment (firefryy) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:66]: they cannot be the same, guava 14 (the version of hive-exec version) is older than all version of google-sloud-storage dependencies, i took the older which is still compatible with it

**Review by lbourgeois — APPROVED** [review]
LGTM

**Review by lbourgeois — DISMISSED** [review]
LGTM

**Comment by lbourgeois** [comment]
build-patch

File: main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml [hunk 2]
```diff
@@ -31,9 +31,9 @@
         </libraryNeeded>
         <libraryNeeded
 		    context="plugin:org.talend.hadoop.distribution.dataproc14"
-		    id="guava-27.0.1-jre"
-		    name="guava-27.0.1-jre.jar"
-		    mvn_uri="mvn:com.google.guava/guava/27.0.1-jre">
+		    id="guava-19.0"
+		    name="guava-19.0.jar"
+		    mvn_uri="mvn:com.google.guava/guava/19.0">
```
Comment (Redwene) [inline main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:36]: cve exist on this jar

**Review by Redwene — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml (hunk 2 above)
Comment (Redwene) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:36]: @lbourgeois

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:36]: @firefryy Can you confirm that this version with CVE is on cluster side ? Maybe they updated it and then we can do the same.

Comment (firefryy) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:36]: yes it's the version on cluster side and if we updated we may have some conflict

**Review by Redwene — APPROVED** [review]

**Comment by AlixMetivier** [comment]
build-patch

**Comment by firefryy** [comment]
build-patch

**Review by Redwene — APPROVED** [review]
LGTM

**Pull request closed without merging**